scheduler/queues/boefje.py

```diff
@@ -11,4 +11,4 @@ def create_hash(self, p_item: models.PrioritizedItem) -> str:
         input_ooi = dict_utils.deep_get(p_item.dict(), ["data", "input_ooi"])
         organization = dict_utils.deep_get(p_item.dict(), ["data", "organization"])
 
-        return mmh3.hash_bytes(f"{boefje_id}-{input_ooi}-{organization}").hex()
+        return mmh3.hash_bytes(f"{input_ooi}-{boefje_id}-{organization}").hex()
```

Comment:
It does feel like we are replicating postgres Index behaviour in python time. Can we not use an index with the Hash type over these fields in the table? Or are we using these hashing in python time for other reasons next to uniqueness?

Reply:
We are indeed using it for more than uniqueness. For a task we need to generate a unique identifier that enables us to check for duplicates. We do it like this because the additional attributes of the task can change (e.g. dispatches attr changes, boefje scan level changes, boefje description changes, etc.) we need a unique combination of attributes of a Boefje/Normalizer task to check if that task is on the queue, or when/how many times it has been processed.

We could use a postgres hashing functionality when persisting the task, however we probably still need a process to generate that unique hash in python-time to create that we can check if a similar task is on the queue or has been processed.

Reply:
ok